source/gui/settingsDialogs.py

```diff
@@ -1492,6 +1493,22 @@ def makeSettings(self, settingsSizer):
 			characterProcessing.CONFIGURABLE_SPEECH_SYMBOL_LEVELS.index(curLevel)
 		)
 
+		self.symbolLevelWord = settingsSizerHelper.addItem(
+			wx.CheckBox(
+				self,
+				# Translators: The label for a setting in the Speech category
+				label=_("Speak all punctuations and symbols when reviewing by &word"),

```

Comment:
```suggestion
				label=_("Speak all punctuation and symbols when reviewing by &word"),
```

or "punctuation marks"